Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -209,39 +208,61 @@ def test_playbooks_results_to_slack_msg(instance_role: str,
                                                    playbook_to_jira_mapping.get(playbook_id)) for playbook_id in failed_tests),
                 "short": False
             }]
-        }]
+        }], True
     title = (f"{title} ({instance_role}) - All Tests Playbooks Passed - Passed:{len(succeeded_tests)}, "
              f"Skipped - {len(skipped_tests)}, Skipped Integrations - {len(skipped_integrations)})")
     return [{
         'fallback': title,
         'color': 'good',
         'title': title,
         'title_link': get_test_report_pipeline_url(pipeline_url),
-    }]
+    }], False
 
 
 def split_results_file(tests_data: str | None) -> list[str]:
     return list(filter(None, tests_data.split('\n'))) if tests_data else []
 
 
-def test_playbooks_results(artifact_folder: Path, pipeline_url: str, title: str) -> list[dict[str, Any]]:
+def get_playbook_tests_data(artifact_folder: Path) -> tuple[set[str], set[str], set[str], set[str]]:
```

Comment:
had a bug: the values in the `*_tests.txt` file might have a `(Mock disabled)` suffix, which i don't need, i just need the playbook id.